apptentive-kit-android/build.gradle

```diff
@@ -0,0 +1,64 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'org.jlleitschuh.gradle.ktlint'
+}
+
+android {
+    compileSdk rootProject.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.minSdkVersion
+        targetSdkVersion rootProject.targetSdkVersion
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+    namespace 'com.apptentive.apptentive_kit_android'
+}
+
+repositories {
+    mavenLocal()
+    maven {
+        url "https://oss.sonatype.org/content/groups/staging/"
+    }
+//
```

Comment:
Should this comment-out part stay?

Reply:
The comments can go. yeah, but will do it with the next one as the change has to come from the private repo. Also, this file isn't a new addition. I am surprised how this wasn't there for all these releases.